docs/config-definition.md

```diff
@@ -25,17 +25,12 @@ metadata:
   name: default
 persistentVolumeProvider:
   aws:
-    region: minio
```

Comment:
Note that I switched this example to AWS because minio PVProvider config is no longer required (or valid), and unfortunately we can't currently mix/match AWS PVProvider with minio backup storage provider due to the single source of credentials.